cmd/analyze.go

```diff
@@ -125,13 +127,17 @@ var analyzeCmd = &cobra.Command{
 		}
 
 		// store to file
-		file, err = os.Create(".harpoon.yml")
-		if err != nil {
-			return fmt.Errorf("failed to create symbols list file: %w", err)
+		if saveAnalysis {
+			file, err = os.Create("harpoon-report.yml")
+			if err != nil {
+				return fmt.Errorf("failed to create symbols list file: %w", err)
+			}
+			mw := io.Writer(file)
+			fmt.Fprintln(mw, symbolsList.String())
```

Comment:
This looks alike a left behind debug

Reply:
It isn't. It's to save the content of the analysis into a file.

Reply:
Oh my bad, I missed the fact it was a Fprint